From e1abc332944fdae5ea99a600b46859324bfd85d1 Mon Sep 17 00:00:00 2001
From: Tobias <ruck.tobias@gmail.com>
Date: Mon, 31 Jan 2022 10:35:07 -0600
Subject: [PATCH 156/196] Bump version to 2.1.3 (#245)

This version supports the NNG interface for allowing easy indexing.
---
 CMakeLists.txt             | 2 +-
 contrib/aur/lotus/PKGBUILD | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 267efda64..76e79c5f8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ set(CMAKE_USER_MAKE_RULES_OVERRIDE
 )
 
 project(lotus
-	VERSION 2.0.3
+	VERSION 2.1.3
 	DESCRIPTION "Lotus is a full node implementation of the Lotus protocol."
 	HOMEPAGE_URL "https://www.givelotus.org"
 )
diff --git a/contrib/aur/lotus/PKGBUILD b/contrib/aur/lotus/PKGBUILD
index 73baec3d5..a48bc84d6 100644
--- a/contrib/aur/lotus/PKGBUILD
+++ b/contrib/aur/lotus/PKGBUILD
@@ -1,7 +1,7 @@
 # Maintainer: Josh Ellithorpe <quest@mac.com>
 
 pkgname=lotusd
-pkgver=2.0.3
+pkgver=2.1.3
 pkgrel=0
 pkgdesc="Lotus with lotusd, lotus-tx, lotus-seeder and lotus-cli"
 arch=('i686' 'x86_64')
-- 
2.39.5 (Apple Git-154)

